samples/client/petstore/typescript-angular-v2/default/api/pet.service.ts

```diff
@@ -338,7 +341,7 @@ export class PetService {
 
         let requestOptions: RequestOptionsArgs = new RequestOptions({
             method: RequestMethod.Get,
-            headers: headers,
+            headers,
```

Comment:
I suggest not to remove the explicit `headers:` part, since without it, refactoring variable names can easily lead to problems

Reply:
I'm not exactly sure hat you mean.  Please elaborate so i can understand.  Thanks!

Reply:
@kenisteward when creating objects like this:
```
{
    something
}
```
instead of
```
{
    something: something
}
```
the name of the property of the object changes when refactoring variable names, leading to potential problems if one does not notice it.

Reply:
Ahh.  Unfortunately, mostly due to how our templates are setup, that will be a big breaking change either way.  

For the 4.3 client and the non 4.3 client they both use the same name for a variable however if someone changed the name for 1 client thinking it would be fine that would also cause errors.  

I'm also assuming you mean refactoring via our mustache templates as one should not be refactoring their code directly as new versions should be automatically generated.